### PR TITLE
Fixed a typo + additional explanatory text proposed

### PR DESCRIPTION
**Pull request by Mafi412**
Vím, že ty uvozovky jsou opravdu marginálie, ale myslel jsem, že když mohu napomoct, aby to mělo nějakou společnou fazónu a bylo to konzistentní, že bych to měl učinit, tak jsem si to dovolil doplnit. (Ani mi za to, prosím, nedávejte community_points, to už bych si přišel vážně blbě... Berte to jako poděkování za Vaši ochotu se nám věnovat při řešení úkolů.)

Ten remark je opravdu jen návrh a je založen na tom, jak jsem pochopil poznámky na stránkách tensorflow u ctc_loss (https://www.tensorflow.org/api_docs/python/tf/nn/ctc_loss), kde psali, cituji: "On TPU and GPU: Only dense padded labels are supported." Takže je možné, že to vůbec není nutné takhle dělat, nezkoušel jsem to. Ale jestli to tak je, tak to dost možná zamezí pár problémům některých studentů. (A opět i u tohoto platí, co jsem psal výše.)

Děkuji.

### CODE REVIEW

**Comment by foxik** [comment]
Dobrý večer,

díky i za opravu uvozovek, ono mi jich tam dost chybí, takže jsem rád, když mě na to někdo upozorní :-) Když nechcete, tak Vám za ně bod nedám ;-)

Nicméně tu druhou poznámku přidávat nebudu -- ona je v tomhle případě dokumentace ne zcela aktuální. Ta `tf.nn.ctc_loss` obsahuje tři různé implementace -- klasickou, pak CuDNN a pak tu "dense", mezi kterými se přepíná právě typem `labels` a tím, jestli je nebo není k dispozici GPU, viz kdyžtak https://github.com/tensorflow/tensorflow/blob/3fac9f1dbb8fc40b4d4691ca7e13f6d3d370f821/tensorflow/python/ops/ctc_ops.py#L935-L973 . Pro GPU je znatelně nejrychlejší varianta právě s těmi sparse labely (ta dense implementace funguje dobře jen když se zkompiluje pomocí `jit_compile`, ale loni to ještě přímočaře nešlo, bylo to funkční jen na TPU).

Já tenhle pull request zavřu a ty backticky opravím sám, ať s tím už nemáte práci. Každopádně díky, i za tu druhou poznámku -- nikdy nevíte, kdy to bude opravdu chyba ;-)
